io_scene_godot/converters/animation/constraint_baking.py

```diff
@@ -86,6 +86,16 @@ def bake_constraint_to_action(blender_object, base_action, bake_type,
             do_object=do_object,
             do_visual_keying=True,
         )
+    else:
+        baked_action = bpy_extras.anim_utils.bake_action(
```

Comment:
I checked [Blender master branch](https://developer.blender.org/diffusion/B/browse/master/release/scripts/modules/bpy_extras/anim_utils.py$32)  they haven't fixed `bake_action` yet. So if other people have same problem as you (got a blender2.8 python API on blender2.79 build) they would still crash here if they don't adding some patch to their blender. Maybe turn some alternative baking functions which works correctly in blender2.8?

Reply:
I [submitted the patch](https://developer.blender.org/D4172) upstream, but I can look at using one of the other functions.

Reply:
Oh sorry, I missed the part that you already submitted the patch.  Thank you for doing that! 

Then maybe we can wait for blender to accept your patch and we keep using `bake_action` here